src/community/blocks.tsx

```diff
@@ -84,22 +84,35 @@ interface SimpleBlockProps {
     image?: RendersNullableElement<typeof Image>;
     description?: string;
     'aria-label'?: string;
+    label?: string;
 }
 
-export const SimpleBlock = ({image, description, 'aria-label': ariaLabel}: SimpleBlockProps): JSX.Element => {
+export const SimpleBlock = ({
+    image,
+    description,
+    'aria-label': ariaLabel,
+    label,
+}: SimpleBlockProps): JSX.Element => {
     return (
         <div aria-label={ariaLabel}>
-            <Inline space={16} alignItems="center">
-                <div
-                    style={applyCssVars({
-                        [mediaStyles.vars.mediaBorderRadius]: vars.borderRadii.mediaSmall,
-                    })}
-                >
-                    {image}
+            <Inline space="between" alignItems="center">
+                <Inline space={16} alignItems="center">
+                    <div
+                        style={applyCssVars({
+                            [mediaStyles.vars.mediaBorderRadius]: vars.borderRadii.mediaSmall,
+                        })}
+                    >
+                        {image}
+                    </div>
+                    <Text2 regular color={vars.colors.textSecondary}>
+                        {description}
+                    </Text2>
+                </Inline>
+                <div className={styles.rightContent}>
+                    <Text2 regular color={vars.colors.brand}>
```

Comment:
```suggestion
                    <Text2 regular color={vars.colors.textBrand}>
```

![image](https://github.com/user-attachments/assets/2aad7bc5-d637-48cb-b042-e704a8bdaf94)